jitify.hpp

```diff
@@ -551,10 +551,12 @@ inline std::vector<std::string> split_string(std::string str,
 static const std::map<std::string, std::string>& get_jitsafe_headers_map();
 
 inline bool load_source(
-    std::string filename, std::map<std::string, std::string>& sources,
+    std::string filename,
+    std::map<std::string, std::string>& sources,
```

Comment:
This needs formatting:
```suggestion
    std::string filename, std::map<std::string, std::string>& sources,
```